sw/host/opentitanlib/src/transport/cw310/mod.rs

```diff
@@ -225,6 +228,14 @@ impl Transport for CW310 {
             Err(TransportError::UnsupportedOperation.into())
         }
     }
+
+    fn jtag(&self, opts: &JtagParams) -> Result<Rc<dyn Jtag>> {
```

Comment:
I'm not sure I understand why this needs to be an `Rc<dyn Jtag>>`, do you have an example of when we need reference counting or what other types might implement `Jtag`?